docs/scanners.md

```diff
@@ -157,6 +157,7 @@ Scanners:
 40032   .htaccess Scanner
 40033	NoSQL Injection MongoDB
 40034   .env File Scanner
+40039   Hidden File Scanner
```

Comment:
Is there a reason to skip to 39?

Reply:
Sorry. It was a typo from my previous pr. Corrected it now.